packages/component-library/src/colorpicker.scss

```diff
@@ -0,0 +1,55 @@
+.react-aria-ColorSwatch {
```

Comment:
We don't use css files in the codebase. For consistency, can you move and declare these on the component file itself?